virgo.html

```diff
@@ -0,0 +1,77 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css" />
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+  <link rel="stylesheet" href="./assets/style.css" />
+  <link rel="stylesheet" href="./assets/reset.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+
+  <title>Virgo</title>
+
+</head>
+
+<body>
+  <!-- Mobile Responsive Nav bar -->
+  <nav>
+    <div class="nav-wrapper">
+
+      <a href="#" data-target="mobile-demo" class="sidenav-trigger"><i class="material-icons">Menu --></i></a>
+      <ul class="left hide-on-med-and-down">
+        <li lass="Home"><a href="index.html">Home</a></li>
+        <li><a class="color-aries" href="aries.html">♈ Aries</a></li>
+        <li><a class="color-taurus" href="taurus.html">♉ Taurus</a></li>
+        <li><a class="color-gemini" href="gemini.html">♉ Gemini</a></li>
+        <li><a class="color-cancer" href="cancer.html">♋ Cancer</a></li>
+        <li><a class="color-leo" href="leo.html">♌ Leo</a></li>
+        <li><a class="color-virgo" href="virgo.html">♍ Virgo</a></li>
+        <li><a class="color-libra" href="libra.html">♎ Libra</a></li>
+        <li><a class="color-scorpio" href="scorpio.html">♏ Scorpio</a></li>
+        <li><a class="color-sagitarius" href="sagitarius.html">♐ Sagittarius</a></li>
+        <li><a class="color-capricorn" href="capricorn.html">♑ Capricorn </a></li>
+        <li><a class="color-aquarius" href="aquarius.html">♒ Aquarius</a></li>
+        <li><a class="color-pisces" href="pisces.html">♓ Pisces</a></li>
+        <li><a class="waves-effect waves-dark btn" href="tarot.html">Tarot</a></li>
+      </ul>
+    </div>
+  </nav>
+  <ul class="sidenav" id="mobile-demo">
+    <li><a href="index.html">Home</a></li>
+    <li><a href="aries.html">Aries</a></li>
+    <li><a href="taurus.html">Tauras</a></li>
+    <li><a href="gemini.html">Gemini</a></li>
+    <li><a href="cancer.html">Cancer</a></li>
+    <li><a href="leo.html">Leo</a></li>
+    <li><a href="virgo.html">Virgo</a></li>
+    <li><a href="libra.html">Libra</a></li>
+    <li><a href="scorpio.html">Scorpio</a></li>
+    <li><a href="sagitarius.html">Sagittarius</a></li>
+    <li><a href="capricorn.html">Capricorn</a></li>
+    <li><a href="aquarius.html">Aquarius</a></li>
+    <li><a href="pisces.html">Pisces</a></li>
+    <li><a class="waves-effect waves-dark btn" href="tarot.html">Tarot</a></li>
+  </ul>
+  <div class="row">
+    <div class="col m7">
+
+    </div>
+    <!-- semantic tag added for better formatting  -->
+    <aside class="col m4 curr-hor daily">
+
+    </aside>
+
+
+    <footer class="footer">
+      <a href="https://icons8.com/icon/2FBe9fua6GC2/aries">Aries icon by Icons8</a>
+    </footer>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Having everything kick off in a `$(document).ready(function() {` usually alleviates the need to declare javascript at the end of the page

It's usually desirable to keep all of your imports in the header and `$(document).ready(function()` allows you to do that